examples/configs/classification_cifar10.yaml

```diff
@@ -43,7 +43,7 @@ data:
           input_dtype: *input_dtype
           train: true
           download: true
-          data_folder: &data_folder ${oc.env:HOME}/.cache/torchok/data/cifar10
+          data_folder: &data_folder ${oc.env:HOME}/cifar10/data
```

Comment:
It's not good to create many folders in user's home directory, one folder is much more convenient for a user

Reply:
As a user, it is not convinient what you suggest. As I can set up as a `oc.env:HOME` anything I want, therefore I can choose where all logs will be

Reply:
Usually, `$HOME` env variable is set by the system since it's a common practice to point to the home directory. A user won't change this variable to have better experience with our library because it introduces a mess...